tasks/components/layout/PatientDetails.tsx

```diff
@@ -65,27 +69,51 @@ export const PatientDetail = ({
   language,
   bedPosition,
   bedsInRoom,
-  patient,
-  onUpdate,
-  onDischarge
+  patient = emptyPatientDetails
 }: PropsWithLanguage<PatientDetailTranslation, PatientDetailProps>) => {
-  const [isShowingConfirmDialog, setIsShowingConfirmDialog] = useState(false)
+  const [isShowingDischargeDialog, setIsShowingDischargeDialog] = useState(false)
   const translation = useTranslation(language, defaultPatientDetailTranslations)
-  const [newPatient, setNewPatient] = useState<PatientDTO>(patient)
+
+  // TODO fetch patient by patient.id replace below
+  const context = useContext(WardOverviewContext)
+
+  const updateMutation = usePatientUpdateMutation(() => undefined)
+  const dischargeMutation = usePatientDischargeMutation(() => context.updateContext({ wardID: context.state.wardID }))
+  const unassignMutation = useUnassignMutation(() => context.updateContext({ wardID: context.state.wardID }))
+  const { data, isError, isLoading } = usePatientDetailsQuery(() => undefined, context.state.patient?.id)
+
+  const [newPatient, setNewPatient] = useState<PatientDetailsDTO>(patient)
   const [newTask, setNewTask] = useState<TaskDTO | undefined>(undefined)
   const [boardObject, setBoardObject] = useState<KanbanBoardObject>({ searchValue: '' })
   const [isShowingSavedNotification, setIsShowingSavedNotification] = useState(false)
-  const [sortedTasks, setSortedTasks] = useState<SortedTasks>({
-    unscheduled: newPatient.tasks.filter(value => value.status === 'unscheduled'),
-    inProgress: newPatient.tasks.filter(value => value.status === 'inProgress'),
-    done: newPatient.tasks.filter(value => value.status === 'done'),
+
+  const sortedTasksByPatient = (patient: PatientDetailsDTO) => ({
```

Comment:
```suggestion
  const sortedTasksByPatient = (patient: PatientDetailsDTO) => ({
```
Pass TaskDTO directly